README.md

```diff
@@ -135,6 +135,18 @@ var limited = qs.parse('a=b&c=d', { parameterLimit: 1 });
 assert.deepEqual(limited, { a: 'b' });
 ```
 
+If you want an error to be thrown whenever the parameter limit is exceeded, set the `throwOnLimitExceeded` option to `true`. This option will generate a descriptive error if the query string exceeds the parameterLimit.
```

Comment:
```suggestion
If you want an error to be thrown whenever the a limit is exceeded (eg, `parameterLimit`, `arrayLimit`), set the `throwOnLimitExceeded` option to `true`. This option will generate a descriptive error if the query string exceeds a configured limit.
```

should we also include `depth` here?

Reply:
Please help me understand about the relation with `depth`.

Reply:
`depth` is also a conceptual limit, that currently when reached, causes a silent behavior change. Throwing would be an alternative.

Reply:
There's already a check for `depth`,

```js
if (options.strictDepth === true) {
    throw new RangeError('Input depth exceeded depth option of ' + options.depth + ' and strictDepth is true');
}
```

Reply:
I am in favor of keeping as is.

Reply:
I suppose since we have strictDepth already, we don't need the functionality in this option.